.gitignore

```diff
@@ -1,4 +1,5 @@
 *.icloud
+.cosine
```

Comment:
What's a `.cosine` file?

Reply:
Cosine is an extension (AI assistant) for VSCode. It creates a local directory `.cosine` to store its vector database and project metadata.